### PR TITLE
LPS-172598 || Refactor on some paths from Forms

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-172598 and related to this PR https://github.com/liferay-forms/liferay-portal/pull/2098

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 1]
```diff
@@ -1764,12 +1764,6 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
-	macro viewPredefinedValueDropdownContent {
-		AssertElementPresent(
-			key_textContent = "${key_textContent}",
-			locator1 = "FormFields#SIDEBAR_ADVANCED_PREDEFINED_SELECT_FROM_LIST_DROPDOWN_CONTENT");
-	}
-
```
Comment (andreefarias) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:1772]: You can use it as one separate commit.

**Comment by andreefarias** [comment]
ci:forward

**Comment by andreefarias** [comment]
Hey @JhossephG,

Could fix it? Something is wrong... run the runPoshiValidation

**Comment by JhossephG** [comment]
> Hey @JhossephG,
> 
> Could fix it? Something is wrong... run the runPoshiValidation

This PR can be used in the future to finish the issue demanded from the Forms modifications, so it's better to keep it that away, I think.

**Comment by epark18** [comment]
@JhossephG Do you know when the Forms modifications will be sent? This should be tested with those changes, so it's best to add your changes to that PR. Forwarding this before the modifications will break some tests in current state.

**Comment by JhossephG** [comment]
@epark18 you can get more context from this PR in this conversation: https://liferay.slack.com/archives/C048YNC5GRK/p1673295684988159

**Comment by epark18** [comment]
@JhossephG Can we close this for now and resend when the Forms changes are ready?

**Pull request closed without merging**